lib/api/apiUtils/authorization/aclChecks.js

```diff
@@ -115,6 +119,16 @@ function isObjAuthorized(bucket, objectMD, requestType, canonicalID) {
             return true;
         }
     }
+
+    // allow public reads on buckets that are whitelisted for anonymous reads
+    // TODO: remove this after bucket policies are implemented
+    const bucketAcl = bucket.getAcl();
+    const allowPublicReads = publicReadBuckets.includes(bucket) &&
```

Comment:
Not sure that works since you seem to compare an instance of a bucket object (`BucketInfo`?) to find in a list of strings.

Reply:
Ah yes, you are right. It's an array though as it's exploded at the top.